Geometry/MTDCommonData/test/testMTDGeometry.py

```diff
@@ -1,6 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
-process = cms.Process("MTDGeometryTest")
+from Configuration.Eras.Era_Phase2C11I13M9_cff import Phase2C11I13M9
+from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
+
+process = cms.Process("MTDGeometryTest",Phase2C11I13M9,dd4hep)
```

Comment:
Why do we need dd4hep here? I think yiou still use DDD test module in this file.

Reply:
@srimanob not really, please see the geometry configuration loaded. This is just dump of the DD4hep test configuration with MTD and the bare containers needed for it

Reply:
Thanks @fabiocos 
Naming is a bit confusion for me.